Core/Libraries/Source/WWVegas/WWLib/wwstring.h

```diff
@@ -553,19 +553,19 @@ StringClass::Get_Buffer (int new_length)
 }
 
 ///////////////////////////////////////////////////////////////////
-//	Peek_Buffer
+//	str (formerly Peek_Buffer)
 ///////////////////////////////////////////////////////////////////
 inline TCHAR *
-StringClass::Peek_Buffer (void)
```

Comment:
There are many functions called "Peek" and most of them return a non-const pointer. To me it looks like this was intentional. Therefore, I suggest to leave non-const `Peek_Buffer` as is.

https://github.com/search?q=repo%3ATheSuperHackers%2FGeneralsGameCode+peek&type=code

Reply:
The equivalent to StringClass::Peek_Buffer is std::string::data

https://en.cppreference.com/w/cpp/string/basic_string/data